src/InterNations/Sniffs/Syntax/MethodTypeHintsSniff.php

```diff
@@ -313,10 +306,9 @@ public function process(CodeSnifferFile $file, $stackPtr)
 
     private function inArray($needle, &$paramDoc)
```

Comment:
inArray() doesn’t sound like it would modify anything but it still does. Can we find a better name?